Bump eui

stacey-gammon opened the pull request.

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6870/)

w33ble commented:
@Stacey-Gammon what version(s) are you planning to target here?

Stacey-Gammon commented:
@w33ble, 6.x and 7.0.  I'm not sure of the protocol, but I have some typings that are in 2.0 that I'd like to work against.  I also added a comment on your PR.

cchaos commented:
These pages (the expanded ones) will need to be updated to fix the breaking changes from `EuiPage`.

<img width="663" alt="screen shot 2018-07-13 at 16 46 39 pm" src="https://user-images.githubusercontent.com/549577/42713468-937a4a42-86bd-11e8-8717-68f5e03fe3be.png">

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6944/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6971/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/6988/)

Stacey-Gammon commented:
homepage looks good:
<img width="1364" alt="screen shot 2018-07-16 at 2 13 05 pm" src="https://user-images.githubusercontent.com/16563603/42775477-67beb09a-8902-11e8-95d9-c05600c0749e.png">

cchaos commented:
@Stacey-Gammon I'm confused, where did my [fixes PR](https://github.com/Stacey-Gammon/kibana/pull/7) go? The changes are not in here and the homepage layout is broke again.

Stacey-Gammon commented:
that's really weird @cchaos!  I have no idea.  Hmmm, let me see if I can remerge the changes.

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/7025/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/7066/)

Stacey-Gammon commented:
well I added your changes and updated the xpack snapshot tests @cchaos but i think now fialing on yet more flaky tests 😭 

```
UI Functional Tests.test/functional/apps/management/_kibana_settings·js.management kibana settings "after all" hook
```

lets find out.

jenkins, test this.

cchaos commented:
Yeah, something about this missing selector `[data-test-subj="advancedSetting-editField-dateFormat:tz"]`...?

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic-kibana-pull-request/7075/)

Stacey-Gammon commented:
Filed the flaky test issue here: https://github.com/elastic/kibana/issues/20893

ready for review

Stacey-Gammon commented:
Ready for final review.  Am out tomorrow - Friday so if I don't get it in today, this will either have to wait till next week or someone else will have to take over.

Stacey-Gammon commented:
@cchaos - is this supposed to happen on dashboard listing page (note vertically centered is new i think):

<img width="1431" alt="screen shot 2018-07-17 at 3 29 20 pm" src="https://user-images.githubusercontent.com/1

Tried to manually review all the other changed snapshots and they look good as far as i can tell.

cchaos commented:
@Stacey-Gammon Yes, that's on purpose [via this line](https://github.com/Stacey-Gammon/kibana/blob/06501dd7d69e4712d0a982f9547649e148b0ce72/src/core_plugins/kibana/public/dashboard/listing/__snapshots__/dashboard_listing.test.js.snap#L44) introduced during the EUI-ification of that page.

snide commented:
Ha. I'll clean that up post this PR @Stacey-Gammon. That line @cchaos is pointing to likely wasn't working (she fixed it!) when I implemented it. I agree it looks goofy :)

Thanks for getting this stuff in.

nreese commented on src/core_plugins/kibana/public/home/components/tutorial/tutorial.js:
```diff
@@ -313,10 +313,15 @@ export class Tutorial extends React.Component {
-        <EuiLink href="#/home">Home</EuiLink> / <EuiLink href="#/home/tutorial_directory">Add Data</EuiLink>
-        <EuiSpacer size="s" />
-        {content}
+          <div>
```
Why is now wrapped in a `div`?

Stacey-Gammon replied:
@cchaos can you answer this? It's part of the code you made that I merged into this PR.

cchaos commented:
@Stacey-Gammon Looks like this should be gtg!

Stacey-Gammon commented:
Great!  I'm on my phone right now, ended with for the day.  Will merge once
I get to a computer or feel free to hit the merge button sooner if it's
blocking anyone.  Not logged in on my phone so can't do it this second.

On Tue, Jul 17, 2018, 6:23 PM Caroline Horn <notifications@github.com>
wrote:

> @Stacey-Gammon <https://github.com/Stacey-Gammon> Looks like this should
> be gtg!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/elastic/kibana/pull/20774#issuecomment-405747731>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APy9kx91FOIJvR_NDjM7Js3_JVj1pZcMks5uHmP1gaJpZM4VPInZ>
> .
>

snide commented:
For the future, reminder to everyone that we need to set dependencies is BOTH package.json and xpack/package.json.

No worries, I missed it to. Have a follow up PR to fix. https://github.com/elastic/kibana/pull/20930

nreese commented:
Lets just take the dependency out of x-pack. That way there is only one version
